NEWS.md

```diff
@@ -76,10 +82,20 @@ Note: We're only listing outstanding class updates.
 
     * `Math.log1p` and `Math.expm1` are added. [[Feature #21527]]
 
-* Socket
+* Method
 
-    * `Socket.tcp` & `TCPSocket.new` accepts `open_timeout` as a keyword argument to specify
-      the timeout for the initial connection. [[Feature #21347]]
+    * `Method#source_location`, `Proc#source_location`, and
+      `UnboundMethod#source_location` now return extended location
+      information with 5 elements: `[path, start_line, start_column,
+      end_line, end_column]`. The previous 2-element format `[path,
+      line]` can still be obtained by calling `#take(2)` on the result.
```

Comment:
```suggestion
      line]` can still be obtained by calling `.take(2)` on the result.
```

We would generally use `Array#take` to refer to an instance method of a specific class, but since this refers to calling a method on the result of the previous method, I think it makes more sense to use `.` instead of `#`.

Reply:
Good catch, thanks!